torch/functional.py

```diff
@@ -1151,5 +1153,115 @@ def _lu_no_infos(A, pivot=True, get_infos=False, out=None):
     func_name='lu')
 lu.__doc__ = _lu_impl.__doc__
 
+def clamp(input, min=None, max=None, out=None):
+    r"""
+    Clamp all elements in :attr:`input` into the range `[` :attr:`min`, :attr:`max` `]` and return
+    a resulting tensor:
+
+    .. math::
+        y_i = \begin{cases}
+            \text{min}_i & \text{if } x_i < \text{min}_i \\
```

Comment:
I think this formula will have to change based on the previous computation for the clamped value, which does allow clamping to produce values less than min.